### DISCUSSION TITLE
Installing Package as Executable

### DISCUSSION THREAD

**Discussion by oraqlle** [vector-of-bool/dds#131] [Q&A] [answered]
I was wondering, does DDS/BPT support building a package into a standalone executable similar to cargo. For example, if you build a CLI tool, you can use DDS to install the package, build it and have access to the produced executable for the tool? 

Thanks

**Comment by vector-of-bool** [comment, marked as answer]
At time of writing, it does not. I do want a feature like this in the future, though. e.g.:

```bash
> bpt getapp somepackage@1.2.3/somepackage:the-app
[ Does work ]
> ls
the-app.exe
```

**Reply by oraqlle** [reply]
Thanks, good to know its in the pipeline.
